### PR TITLE
feat: Add LH status to messages (WEBAPP-6266)

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: src/script/assets/AssetService.ts [hunk 1]
```diff
@@ -39,7 +39,8 @@ export interface CompressedImage {
 }
 
 export interface AssetUploadOptions {
-  expectsReadConfirmation: boolean;
+  expectsReadConfirmation?: boolean;
```
Comment (bennycode) [inline src/script/assets/AssetService.ts]: So far `expectsReadConfirmation` has been mandatory. Why do you want to make it optional? Isn't it good to have it always being `true` or `false`?

Comment (ffflorian) [reply src/script/assets/AssetService.ts]: The default will be falsy, so we can omit it, too. Just wanted to make the API easier :slightly_smiling_face:

Comment (bennycode) [reply src/script/assets/AssetService.ts]: I would rather have a pure `false` than working with falsy to be explicit.

Comment (ffflorian) [reply src/script/assets/AssetService.ts]: Done!

**Pull request merged**